src/Lorentz/Contracts/BaseDAO/FA2.hs

```diff
@@ -129,5 +353,5 @@ assertNeq0or1 = do
     dup
     push @Natural 1
     if IsEq
-    then failUsing [mt|FA2_OPERATION_PROHIBITED|]
-    else failUsing [mt|FA2_TOKEN_UNDEFINED|]
+    then failUsing [mt|fA2_OPERATION_PROHIBITED|]
```

Comment:
Here we seem to fail on both branches of the conditional, which seems wrong to me. Also I am not sure there is a `fA2_OPERATION_PROHIBITED` error in FA2 spec. Also, since all tests pass now, it seems that we have some uncovered cases in tests.

Reply:
This one is custom. The spec doesn't say anything about error in case there is a `tId` equal 1 in `updateOperators` entrypoint

Reply:
If it is custom, then I think it shouldn't be prefixed with `fA2`.  Also,

> Here we seem to fail on both branches of the conditional, which seems wrong to me.
> Since all tests pass now, it seems that we have some uncovered cases in tests.

Reply:
Maybe here we can fail with the same `#fROZEN_TOKEN_NOT_TRANSFERABLE` since it actually the same situation

Reply:
Or maybe there would be better to change this exception for something like `#fROZEN_TOKEN_OPERATION_PERMITED` to use it everywhere, where operation can't be done with frozen token